doc/ReleaseNotes/_ReleaseNotes.md

```diff
@@ -137,6 +137,7 @@
  * [iOS] Add fail-safe on `FrameworkElement.WillMoveToSuperview` log to `Application.Current.UnhandledException`
  * [Wasm] Fixed `System.UriConverter` is being linked out
  * 145075 [Android] [Wasm] Android and Wasm don't match all specific UWP behaviors for the Image control.
+ * 146648 [Android] fixed ListView grouped items corruption on scroll
```

Comment:
No need to undo for now. but for canary fixes we don't need to mark in the release notes.